### PR TITLE
v1.0.0 release candidate 1

### PR DESCRIPTION
**Pull request by generictjohnson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,13 @@ b2sdk>=0.0.0,<1.0.0
 
 # Release History
 
-## 0.1.10 (Not released yet)
+## 1.0.0 (not released yet)
```
Comment (ppolewicz) [inline README.md]: 1.0.0-rc1

File: README.md [hunk 2]
```diff
@@ -33,7 +33,13 @@ b2sdk>=0.0.0,<1.0.0
...
+
+Changes:
+
+* Deprecate some transitional method names to version 0 in preparation for v1.0.0.
```
Comment (ppolewicz) [inline README.md]:
```suggestion
* Deprecate some transitional method names to v0 in preparation for v1.0.0.
```

File: README.md (hunk 1 above)
Comment (generictjohnson) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (generictjohnson) [reply README.md]: Done.

**Pull request merged**